src/modules/launchbox/entities/launchbox.entity.ts

```diff
@@ -241,8 +241,6 @@ export class LaunchboxTokenLeaderboard {
   name: 'launchbox_token_leaderboard_actions',
 })
 export class TokenConfiguredAction {
-  @Exclude()
```

Comment:
@noelukwa Why did you remove this ?

Reply:
it does not do work as intended

Reply:
> it does not work as intended

Can you confirm it dosen't return the _id to the client ?